src/containers/CreatePipelineRun/CreatePipelineRun.test.js

```diff
@@ -488,32 +528,6 @@ describe('CreatePipelineRun', () => {
     expect(getByPlaceholderText(/select serviceaccount/i)).toBeTruthy();
   });
 
-  it('resets Pipeline and ServiceAccount when namespace changes controlled', async () => {
```

Comment:
Who don't we want it reset in this situation anymore?

Reply:
We do, but there's no difference in behaviour anymore between controlled / uncontrolled for this scenario so this test was adding no additional value. It's already covered by the test above it.